src/opensearch.py

```diff
@@ -118,7 +118,7 @@ def install(self):
         """Temporary (will be deleted later) - Download and Un-tar the opensearch distro."""
         try:
             response = requests.get(
-                "https://artifacts.opensearch.org/releases/bundle/opensearch/2.4.0/opensearch-2.4.0-linux-x64.tar.gz"
+                "https://artifacts.opensearch.org/releases/bundle/opensearch/2.4.1/opensearch-2.4.1-linux-x64.tar.gz"
```

Comment:
Honestly, I want to. But given the extracted size of `2.5` (~1Gb - compared to ~800Mb for `2.4.1`) and the limitations on the GH runners we're facing - I'd prefer to postpone this upgrade until we figure out exactly which limit we're hitting on the runners.

Reply:
Do we really want to hold back product features because we're worried our infrastructure can't handle it? This seems like a concern worth raising to management. 

According to [the GH actions docs](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources), we have 14gb to play with. Considering we've got Carl's cool caching coming in soon, and it's only a diff of 200mb, I think it's worth trying sometime soon.

Reply:
@deusebio - yes I'm aware, but this is coming from when we didn't have a snap yet. We are tracking this on [DPE-1261](https://warthogs.atlassian.net/browse/DPE-1261) - and are planning on replacing it this sprint.